site/content/getting-started/_index.md

```diff
@@ -163,19 +169,21 @@ kubectl patch ingress httpbin -p '{"spec":{"ingressClassName": "contour"}}'
 
 Now we're ready to send some traffic to our sample application, via Contour & Envoy.
```

Comment:
maybe a future change would be to do this via the Gateway API example modified in this PR as well, but probably requires a little refactor of the flow of this doc

Reply:
Yeah actually what's interesting about how it is right now is that even if we're using the Gateway provisioner to provision Contour/Envoy, we still use Ingress to define routing rules. I should call that out a bit more explicitly, to make it clear that using the Gateway provisioner for provisioning does not mean you have to use HTTPRoute/TLSRoute for defining routes. If we did want to use `HTTPRoute`, then yeah we'd have to split up some of the "Test it out" section depending on the installation method.